tests/syft/ast/functionality_test.py

```diff
@@ -112,6 +112,7 @@ def test_len(custom_client: Client) -> None:
     assert result == result_from_ptr
 
 
+@pytest.mark.xfail(strict=False)
 def test_iter(custom_client: Client) -> None:
```

Comment:
Q: Why does this function depend on `static_attribute` and not length? 

```
assert len(a) == a_ptr.__len__()
    
    for _ in range(0, len(a)):
        print(_)
        assert next(iter_from_ptr).get() == next(iter_from_obj)
```

Reply:
I think it was a mistake when devising the test.